WalletWasabi/Blockchain/TransactionBroadcasting/TransactionBroadcaster.cs

```diff
@@ -93,6 +98,58 @@ public async Task<BroadcastingResult> BroadcastAsync(SmartTransaction tx, Cancel
 	}
 }
 
+public class ThirdPartyTransactionBroadcaster : IBroadcaster
+{
+	public static readonly ImmutableArray<ThirdPartyBroadcasterInfo> Providers =
+	[
+		new("BlockstreamInfo", ("https://blockstream.info", "http://explorerzydxu5ecjrkwceayqybizmpjjznk5izmitf2modhcusuqlid.onion"), "/api/tx"),
+		new("MempoolSpace", ("https://mempool.space", "http://mempoolhqx4isw62xs7abwphsq7ldayuidyx2v2oethdhhj6mlo2r6ad.onion"), "/api/tx")
+	];
+
+	public ThirdPartyTransactionBroadcaster(string providerName, Network network, IHttpClientFactory httpClientFactory)
+	{
+		SelectedBroadcaster = Providers.FirstOrDefault(x => x.Name.Equals(providerName, StringComparison.InvariantCultureIgnoreCase)) ?? throw new NotSupportedException($"Transaction broadcaster '{providerName}' is not supported");
+		Network = network;
```

Comment:
The `Network` is just a parameter for the provider filter. It shouldn't be a property. The problem is that we have the same endpoints for all networks and that's not okay. Note that in fee rate and exchange rate providers we don't care but the network but here is different because using the wrong network will fail.

You have to have a `Network` field as part of the `ThirdPartyBroadcasterInfo`. Btw, I removed all code called `ThirdParty` and now they are call `External`.